app/views/sequences/show.html.erb

```diff
@@ -16,6 +16,8 @@
           <li class="active"><a href="#lineage-tree" id="lineage-tree-tab" data-toggle="tab">Lineage tree <span class="small glyphicon glyphicon-question-sign help btn-icon"></span></a></li>
           <li><a href="#lineage-table" id="lineage-table-tab" data-toggle="tab">Lineage table <span class="small glyphicon glyphicon-question-sign help btn-icon"></span></a></li>
           <li><a href="#proteins" data-toggle="tab">Protein matches <span class="badge"><%= @entries.size %></span></a></li>
+          <li class="active"><a href="#ec-tree" id="ec-tree-tab" data-toggle="tab">EC number tree <span class="small glyphicon glyphicon-question-sign help btn-icon"></span></a></li>
```

Comment:
There can only be one active tab at a time, remove `class="active"`. 


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3385) by @bmesuere on Wed Oct 21 2015 at 16:50._